SnoopCompileCore/src/snoopi.jl

```diff
@@ -99,5 +167,11 @@ function __init__()
     end
     precompile(start_timing, ())
     precompile(stop_timing, ())
+
+    @assert precompile(Core.Compiler.Timings.reset_timings, ())
```

Comment:
Are these actually required? Unlike `@snoopi` you're not calling `jl_set_typeinf_func`, right?

Reply:
`@snoopi` rips out the function that's currently being used to infer code and replaces it with something new. Unfortunately, if in the "in between" state you need to run inference, you're hosed. So precompilation is used to make sure that never happens. Try deleting those from `@snoopi` and see what happens (I don't remember the answer anymore, but I am almost certain there is some horrible,session-catastrophic error), then try the same thing in your PR.

Reply:
Oh, I see! :) Thanks. I assumed they were there so that these functions don't show up in trace that's being recorded. That makes sense! I'll see what happens if I delete them. 👍

Reply:
Oh, good point. If they are annoying then they are fine to precompile. Since they were initially in the same place as the others I thought you had a different purpose.